lambda/interop/model.go

```diff
@@ -18,10 +19,10 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+var MaxPayloadSize int = 6*1024*1024 + 100 // 6 MiB + 100 bytes
```

Comment:
annoying that we cannot use a single type due to mixed usages 😢 (server.go requires an `int`; not `int64`)